dkc/core/tests/conftest.py

```diff
@@ -6,10 +6,17 @@
 
 
 @pytest.fixture
-def api_client():
+def api_client() -> APIClient:
     return APIClient()
 
 
+@pytest.fixture
+def authenticated_api_client(user) -> APIClient:
```

Comment:
I think this will have limited usefulness, since in most test cases we'd want a specific user that is linked via permissions to other documents, rather than an arbitrary user. But it's at least useful in some cases.

Reply:
Until we have permissions, I think it will be useful. It may also be helpful for any testing of `create` endpoints, where the user doesn't have existing permissions on anything. I'd be fine with removing it later if it becomes obsolete.

Reply:
I've removed this for now.